include/tvm/runtime/packed_func.h

```diff
@@ -45,6 +45,15 @@
 #define TVM_RUNTIME_HEADER_ONLY 0
 #endif
 
+// Always inline macro only use in template
+// expansion cases where we know inline is important.
+#ifdef _MSC_VER
+#define TVM_ALWAYS_INLINE __forceinline inline
+#pragma warning(disable : 4068)
```

Comment:
Just curious, which warning are we disabling here?

Reply:
It seems to be irrelevant here, i just removed it